packages/react/src/components/Layer/Layer.base.tsx

```diff
@@ -54,7 +54,19 @@ export const LayerBase: React.FunctionComponent<ILayerProps> = React.forwardRef<
         return doc?.getElementById(hostId) ?? null;
       } else {
         const defaultHostSelector = getDefaultTarget();
-        return (defaultHostSelector ? doc?.querySelector(defaultHostSelector) : doc?.body) ?? null;
+
+        let host = defaultHostSelector && doc?.querySelector(defaultHostSelector);
```

Comment:
I suggest defining `exports` for `createDefaultLayerHost` and `cleanupDefaultLayerHost`.

Reply:
Updating incoming.

Reply:
done.